sdv/constraints/base.py

```diff
@@ -21,6 +21,11 @@ def _get_qualified_name(obj):
     return module + '.' + obj_name
 
 
+def _is_function(obj):
+    """Return true if the object is a function (not including lambdas)."""
+    return callable(obj) and obj.__name__ != (lambda: None).__name__
```

Comment:
If the function passed is the result of another function, then the problem still occurs. Not sure of a way to identify that scenario. ie.

```
def function_creator():
    def func(x):
        return x + 4
 ColumnFormula(
    column='column',
    formula=function_creator(),
    handling_strategy='transform'
)